sklearn/svm/_base.py

```diff
@@ -335,7 +332,7 @@ def _dense_fit(self, X, y, sample_weight, solver_type, kernel, random_seed):
             y,
             svm_type=solver_type,
             sample_weight=sample_weight,
-            class_weight=self.class_weight_,
+            class_weight=getattr(self, "_class_weight", np.empty(0)),
```

Comment:
I think there is a `TODO(1.4)` here to change `_class_weight` to `class_weight_`?

Same comment for the other locations.

Reply:
I think `_class_weight` was used to avoid the depreciation warning?

Reply:
Indeed, using `getattr(self, "class_weight_", np.empty(0))` results in a `FutureWarning`.